### PR TITLE
Register page & Decline invitations

### PR DESCRIPTION
**Pull request by qgerome**
- Create a new page to register an account using an invitation token.
- List all pending invitations on the account page
- Display a badge like in the menu when there are pending invitations

### CODE REVIEW

**Review by pvanliefland — APPROVED** [review]
Code looks 👍, will test the UX further next year (🥁🙄)

File: src/core/helpers/page.ts [hunk 1]
```diff
@@ -83,6 +84,7 @@ export function createGetServerSideProps(options: CreateGetServerSideProps) {
         !features.some((f) => f.code === "workspaces") &&
         ctx.resolvedUrl.startsWith("/workspaces")
       ) {
+        console.log("SECOND");
```
Comment (pvanliefland) [inline src/core/helpers/page.ts:87]: (leftover)

**Review by pvanliefland — CHANGES_REQUESTED** [review]
Nice, just a `console.log` to remove

File: src/core/helpers/page.ts (hunk 1 above)
Comment (qgerome) [reply src/core/helpers/page.ts]: done

**Pull request merged**